package.json

```diff
@@ -14,7 +14,8 @@
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.22.3",
     "react-scripts": "5.0.1",
-    "typescript": "^5.4.3",
+    "react-scroll-to-top": "^3.0.0",
```

Comment:
package-lock.json should not be a part of this change. put it in .gitignore. Only package.json changes should be good

Reply:
Sir, I added package-lock.json in  .gitignore but it is not getting ignored